include/Customer.h

```diff
@@ -1,19 +1,41 @@
 #pragma once
 
+//** KOOS - I would probably encourage the use of library instead of source
 #include <json.hpp>
 
 /*
 define customer type
 */
-typedef struct Customer {
+// typedef struct Customer final {
+
+//** KOOS - make non inheritable objects as final
+//** and no typdef.. this is C style and not needed in C++
+
+struct Customer final {
 
     std::string lname;
     std::string fname;
     std::string email;
     int id;
     float spent;
+    
+//** KOOS:    
+//** < operator overload for sorting
+    bool operator < (const Customer &compare_cust) const 
+    {
+        return (id < compare_cust.id);
+    }
 
-} Customer;
+//** KOOS:    
+//** > operator overload for sorting.
```

Comment:
I thought about doing this (actually changed it twice) but decided it was simpler in this case to make the function for sort.

Reply:
So -- I'm on board with your comment; I figured it could have gone either way.